packages/web/components/CurateSpecialties/CurateSpecialties.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { useModal } from '../../hooks';
+
+const CurateSpecialties = ({ data = [] }) => {
+	const { t } = useTranslation(['profile']);
+	const { openModal } = useModal();
+
+	const getChildrenCategory = (category) => {
+		const filteredCategory = data.filter((c) => c.parent_id === category);
+		return filteredCategory.length > 0 ? filteredCategory[0].term : 'Todas';
+	};
+
+	return (
+		<Container>
+			<h3>{t('profile:curateMySpecialties')}</h3>
+			<List>
+				{data.length === 0 ? (
```

Comment:
```suggestion
				{data.length === 0 ? (
```

Aqui você pode inverter a lógica para `{data.length ? truthyValue : falsyValue }` obviamente precisar inverter o código.

Reply:
Feito!